build/rstudio-base.def

```diff
@@ -0,0 +1,2 @@
+Bootstrap: docker
+From: rocker/rstudio:4.4.2
```

Comment:
I saw the docker hub link, but might be helpful to note the rocker project homepage as well: https://rocker-project.org/